catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/SearchFormsApplication.java

```diff
@@ -107,6 +111,34 @@ public void init() {
                 .filter(Objects::nonNull)
                 .collect(Collectors.toList()),
         MAPPER::toJson);
+
+    /** Gets me all query templates that are shared with me * */
+    // TODO: This is no longer required, incorporate client-side filtering using new adjustments to
+    // Template Transformer
+    get(
+        "/forms/sharing",
+        (req, res) -> {
+          String currentUser = SubjectUtils.getEmailAddress(SecurityUtils.getSubject());
+
+          return util.getMetacardsByFilter(QUERY_TEMPLATE_TAG)
+              .entrySet()
+              .stream()
+              .map(Map.Entry::getValue)
```

Comment:
✏️ Since we're just using the key to get the value, we could stream the result of `values()` instead of `entrySet()`